openshift/ftest/src/test/resources/arquillian.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
 	<extension qualifier="openshift">
-		<property name="definitionsFile">src/test/resources/hello_pod.json</property>
+		<property name="env.config.url">src/test/resources/hello_pod.json</property>
```

Comment:
Ok then I wait for @dipak-pawar to decide if we close it or not. For now the other PR is already merged.